src/content/docs/victory-histogram.md

```diff
@@ -15,11 +15,10 @@ scope:
 
 VictoryHistogram renders a dataset as series of bars representing "bins", allowing you to see the distribution of your data. The data you pass in will be "binned" according to the `bin` prop you pass, allowing for flexibility in how these bins are determined. VictoryHistogram can be composed with [`VictoryChart`][] to create histogram charts.
 
-Because each bar represents a bin rather than a particular data point (like you would have with `VictoryScatter` for example), when accessing `datum` via a prop that passes `datum` such as `style`, datum will have properties `x`, `x0`, `x1`, `y`, and `binnedData`. `x` is the midpoint between the bin, `x0` is the beginning of the bin, `x1` is the end of the bin, `y` is the aggregate amount of data points within that bin, and `binnedData` is an array of the original data points that were grouped into this bin.
+Compared to [`VictoryBar`][], which is good for plotting quantitative data, if you are looking to plot data that is qualitative, then use `VictoryHistogram`.
```

Comment:
This is backwards, right? VictoryHistogram doesn't work with qualitative data

Reply:
nitpick: try to avoid "you" in technical writing. Also idiomatic phrases like "looking to plot data" may confuse users who don't have english as a first language. Maybe "`VictoryHistogram` is intended to  be used with quantitative data. Please use `VictoryBar` for qualitative or categorical data"

Reply:
Got it! Yes I mixed those two up, but I've fixed that now and removed other usages of "you" throughout